tests/test_write_docstring_to_file.py

```diff
@@ -0,0 +1,77 @@
+import os
+from tempfile import NamedTemporaryFile, TemporaryDirectory
+import pytest
+from fml_doc_gen.write_docstring_to_file import write_docstring_to_file
+
+
+@pytest.fixture
+def valid_docstring():
+    """Fixture for a valid docstring."""
+    return """
+    Writes the generated docstring to a specified output file.
+
+    Parameters
+    ----------
+    docstring : str
+        The docstring to be written to the file.
+    output_file : str
+        The path to the output file.
+
+    Returns
+    -------
+    None
+        This function does not return anything.
+    """
+
+@pytest.fixture
+def empty_docstring():
+    """Fixture for an empty docstring."""
+    return "   "
+
+def test_print_only(valid_docstring, capsys):
```

Comment:
I think we don't need this test, but damn I didnt know you could do that. Good stuff though